consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/messagewrappers/NewRound.java

```diff
@@ -24,10 +29,31 @@ public NewRound(final SignedData<NewRoundPayload> payload) {
   }
 
   public RoundChangeCertificate getRoundChangeCertificate() {
-    return getSignedPayload().getPayload().getRoundChangeCertificate();
+    return getPayload().getRoundChangeCertificate();
   }
 
   public SignedData<ProposalPayload> getProposalPayload() {
-    return getSignedPayload().getPayload().getProposalPayload();
+    return getPayload().getProposalPayload();
+  }
+
+  public Block getBlock() {
+    return getProposalPayload().getPayload().getBlock();
+  }
+
+  @Override
+  public BytesValue encode() {
+    final BytesValueRLPOutput rlpOut = new BytesValueRLPOutput();
+    rlpOut.startList();
```

Comment:
Why are these encoded inside an additional list whilst some others also with a single e.g. Commit are writing the output as is?